File: practices/time.py
#SP 6th Time of day 

import datetime

current = datetime.datetime.now()
hour = current.hour

if hour >= 18:
    print(f"Hello the time is {hour}, good evening!")
elif hour > 12 and hour < 18:
    print(f"Hello The time is {hour}, good afternoon!")
elif hour >= 6 and hour < 12:
    print(f"Hello the time is {hour}, good morning!")

File: c_programming/practices/time.c
//SP 6th Time of Day

#include <stdio.h>

int main(void){
    int time;

    printf("Hello user, please type the hour in military time:\n ");
    scanf("%d", &time);

    if(time >= 18 && time <= 24){
        printf("Hello the time is %d, Good Evening!\n", time);
    }else if(time >= 12 && time < 18){
        printf("Hello the time is %d, Good Afternoon!\n", time);
    }else if(time >= 1 && time < 12){
        printf("Hello the time is %d, Good Morning\n!", time);
    }else{
        printf("PLease type a actual hour!\n");
    }

    return 0;
}